TweetRelated/sampling.py

```diff
@@ -0,0 +1,115 @@
+import math
+import numpy as np
+import matplotlib.pyplot as plt
+
+plt.rcParams['font.size'] = 14
+plt.rcParams['legend.fontsize'] = 'medium'
+
+'''
+Read txt to numpy arrays
+'''
+
+
+def txt_to_matrix(filename):
+    try:
+        file = open(filename)
+    except IOError:
+        print("File doesn't exist: " + filename)
+        return None
+    else:
+        lines = file.readlines()
+        rows = len(lines)
+
+        datamat = np.zeros((rows, 1), dtype=int)
+
+        row = 0
+        for line in lines:
+            line = line.strip().split('\t')
+            datamat[row, :] = line[:]
+            row += 1
+
+    return datamat
+
+
+'''
+Sample by days
+'''
+hr = "/Volumes/TOSHIBA SSD/PycharmProjects/COVID-19/COVID-19-TweetIDs-master/2020-"
+hw = "/Users/mashazhou/PycharmProjects/COVID-19/COVID-19-TweetIDs-samples/"
```

Comment:
Remove using hr and hw. Save required file in parent directory of project. So that any developer who runs it can run the code successfully.